teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @specialization = specialization
+  end
+
```

Comment:
- [ ] I get the same error as in Student defined above when I try to instantiate a Teacher object. Kindly fix this.

  ![mmesay-teac](https://user-images.githubusercontent.com/36542986/145906725-d71dd652-32e9-4e69-b800-5f12f76127b6.jpg)